anta/tests/logging.py

```diff
@@ -433,7 +433,7 @@ def test(self) -> None:
         if len(command_output) == 0:
             self.result.is_success()
         else:
-            self.result.is_failure("Device has reported syslog messages with a severity of ERRORS or higher")
+            self.result.is_failure(f"Device has reported syslog messages at threshold level or above:\n{command_output}")
```

Comment:
Please keep the ERROR in the message to make it clear when reading the info (otherwise I have to look at the test ddescription ina ddition)

```suggestion
            self.result.is_failure(f"Device has reported syslog messages with a severity of ERRORS or higher:\n{command_output}")
```

Reply:
reverted. thanks!!